### PR TITLE
(SUP-2500) Remove Harmful terminology in Readme and Manifests

### PR DESCRIPTION
**Pull request by henrywangpuppet**
Updated the Harmful Terminology in the Read and Manifests file per:

blacklist > blocklist
compile master > compiler
high availability > disaster recovery
master of masters > primary server
master > primary server
master (branch) > main (branch)
monolithic > standard 
whitelist > allowlist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ If you are configuring the Dashboard node via a `puppet apply` workflow, you wil
 
 ## Usage
 
-### Configure a Monolithic Master and a Dashboard node
+### Configure a Standard Primary Server and a Dashboard node
 
 ```puppet
 node 'master.example.com' {
```
Comment (MartyEwings) [inline README.md:100]:
```suggestion
node 'primary.example.com' {
```

File: README.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ puppet_metrics_dashboard::postgres_host_list:
   - "postgres02.example.com"
 ```
 
-### Configure Master, Compiler running PuppetDB and a Dashboard node
+### Configure Primary Server, Compiler running PuppetDB and a Dashboard node
 
 ```puppet
 node 'master.example.com' {
```
Comment (MartyEwings) [inline README.md:171]:
```suggestion
node 'primary.example.com' {
```

File: README.md [hunk 3]
```diff
@@ -241,7 +241,7 @@ The `--pattern` flag accepts a Ruby glob argument, which the script will interna
 
 ### Allow Telegraf to access PE-PostgreSQL
 
-The following class is required to be applied to the Master (or the PE Database node if using external PostgreSQL) for collection of PostgreSQL metrics via Telegraf.
+The following class is required to be applied to the Primary Server (or the PE Database node if using external PostgreSQL) for collection of PostgreSQL metrics via Telegraf.
 
 ```puppet
 node 'master.example.com' {
```
Comment (MartyEwings) [inline README.md:247]: There are several instances of master being used as a sample node name, i would find and change all of these as well

**Review by MartyEwings — COMMENTED** [review]
Few instances of master being used as a sample note fqdn that need sought out and changed as well

**Comment by genebean** [comment]
> Updated the Harmful Terminology in the Read and Manifests file per:
> 
> high availability > disaster recovery

While I agree that DR more accurately represents what PE provides, I have two problems with this:
1. HA is not equal to DR so they are not interchangable
2. HA is not harmful terminology

**Review by genebean — COMMENTED** [review]
Existing changes look good, it just needs the modifications already suggerted by @MartyEwings and needs the PR's comment itself corrected as I noted in another comment.

File: README.md (hunk 1 above)
Comment (henrywangpuppet) [reply README.md]: Updated

**Review by MartyEwings — COMMENTED** [review]
@henrywangpuppet looks good to me if you rebase down to 1 commit ill merge

**Comment by henrywangpuppet** [comment]
@MartyEwings The commits have been combined into 1 commit. Hopefully, I did it right this time. :D

**Pull request merged**